com.prateekmedia.appimagepool.appdata.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.prateekmedia.appimagepool</id>
```

Comment:
The domain `prateekmedia.com` doesn' t seem to exist. So this app-id is invalid.

Reply:
I changed it to com.github.prateekmedia.appimagepool

Reply:
we prefer `io.github.*` as it is the domain for github pages.